.travis.yml

```diff
@@ -18,9 +18,10 @@ install:
 rvm:
   - 2.0.0
   - 2.1.10
-  - 2.2.7
-  - 2.3.4
-  - 2.4.1
+  - 2.2.10
+  - 2.3.7
+  - 2.4.4
+  - 2.5.1
```

Comment:
Can we make these `2.5.x` etc, so that they stay up-to-date automatically?

Reply:
Do you like the automation only for 2.5.x? how about older Ruby 2.4.x, 2.3.x and etc.?

Reply:
You were saying "these 2.5.x etc". okay. Rebasing.